clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -346,4 +344,52 @@ public AlterConfigsResult alterConfigs(Map<ConfigResource, Config> configs) {
      * @return                The AlterConfigsResult
      */
     public abstract AlterConfigsResult alterConfigs(Map<ConfigResource, Config> configs, AlterConfigsOptions options);
+
+    /**
+     * Query the log directory information for the specified log directories.
+     * All log directories on a broker are queried if an empty collection of log directories is specified for this broker
```

Comment:
Sure. I replaced this comment with `Query the information of all log directories on the given set of brokers`.